query-node/mappings/src/content/commentAndReaction.ts

```diff
@@ -172,6 +172,7 @@ export function setReactionsCount(
     reactionsCountByReactionId.updatedAt = eventTime
 
     ++entity.reactionsCount
+    if (entity instanceof Comment) ++entity.reactionsAndRepliesCount
```

Comment:
Please don't use one-line `if`s. While it is a valid construct, it makes the code harder to read. 
```suggestion
    if (entity instanceof Comment) {
      ++entity.reactionsAndRepliesCount
    }
```